internal/workload/podspec_updates.go

```diff
@@ -44,6 +44,10 @@ const (
 	// DefaultHealthCheckPort is the used by the proxy to expose prometheus
 	// and kubernetes health checks.
 	DefaultHealthCheckPort int32 = 9801
+
+	// DefaultAdminPort is the used by the proxy to expose prometheus
+	// and kubernetes health checks.
+	DefaultAdminPort int32 = 9802
```

Comment:
I think this can go since it's a duplicate of the field above?

Reply:
Fixed.